collect_app/src/test/java/org/odk/collect/android/formmanagement/FormMetadataParserTest.java

```diff
@@ -122,6 +122,27 @@ public void cleansUpReferenceManager() throws Exception {
         }
     }
 
+    @Test
+    @SuppressWarnings("PMD.JUnitUseExpected")
+    public void cleansUpReferenceManagerAfterFail() throws Exception {
+        File formXml = File.createTempFile("form", ".xml");
+
+        FormMetadataParser formMetadataParser = new FormMetadataParser(referenceManager);
+
+        try {
+            formMetadataParser.parse(formXml, mediaDir);
```

Comment:
Should be a `fail` at the end of the `try` I think. Otherwise, we could end up with a confusing failure later.